benches/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "benches"
+edition = "2024"
+license.workspace = true
+version.workspace = true
+
+[[bin]]
+name = "streaming"
+path = "src/streaming.rs"
+
+[dependencies]
+anyhow.workspace = true
+clap = { version = "4.5.49", features = ["derive"] }
+futures.workspace = true
+indicatif = "0.18.0"
+parking_lot.workspace = true
+rlimit = "0.10.2"
```

Comment:
didn't know about this crate, we should replace `fdlimit` with it if its more up-to-date/maintained. do you mind adding an issue to the dev board for that if not in this pr?

Reply:
Yeah I saw fdlimit in the benches. I can make one.